packages/input/package.json

```diff
@@ -5,18 +5,21 @@
   "author": "chef <serhii.havrylenko@gmail.com>",
   "homepage": "https://github.com/serhii-havrylenko/monorepo-babel-ts-lerna-starter",
   "license": "MIT",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
```

Comment:
Why path to types is dropped?

Reply:
leftover from trying to execute that somehow
restored